src/Qir/Controller/Tests.QirController/QirDriverGeneratorTests.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
```suggestion
// Copyright (c) Microsoft Corporation.
```

Reply:
nit: files in this repo no longer have the "All rights reserved" portion of the header.